packages/eslint-plugin-template/tests/rules/i18n/cases.ts

```diff
@@ -173,6 +209,20 @@ export const invalid = [
            ~~~~~~~
     `,
   }),
+  // Not working 🤷🏼‍♂️
+  // convertAnnotatedSourceToFailureCase({
+  //   description: 'should fail if `i18n-*` attribute is missing on `Template`',
+  //   annotatedSource: `
+  //     <ng-template dir="Hello"></ng-template>
+  //                  ~~~
+  //   `,
+  //   messageId: i18nAttribute,
+  //   data: { attributeName: 'dir' },
+  //   annotatedOutput: `
+  //     <ng-template dir="Hello" i18n-dir></ng-template>
+  //                  ~~~
+  //   `,
+  // }),
```

Comment:
I'm having problems with this and other commented tests. I don't know if I'm missing something, but apparently attributes in `ng-template` aren't parsing correctly.

This is the result from `ASTExplorer`:

<img width="1301" alt="Template parser with attributes" src="https://user-images.githubusercontent.com/11965907/134427462-017bc929-c875-4509-9afd-2cf98421a326.png">

Note that the `attribute` doesn't come as `TextAttribute` as usual. I've tested with `Angular` and this is the result:

<img width="1302" alt="Angular with attributes" src="https://user-images.githubusercontent.com/11965907/134427557-c61a8e1d-d308-404d-ad19-d71d38430ee3.png">

Reply:
@rafaelss95 I looked in to this just now and I think it was just a case of missing visitor keys. I have addressed it here: https://github.com/angular-eslint/angular-eslint/pull/968